indexer/modules/custom/uniswap_v3/uniswap_v3_pool_job.py

```diff
@@ -237,32 +222,190 @@ def get_exist_pools(db_service, position_token_address):
     try:
         result = (
             session.query(UniswapV3Pools)
-            .filter(UniswapV3Pools.position_token_address == hex_str_to_bytes(position_token_address))
+            .filter(UniswapV3Pools.position_token_address == bytes.fromhex(position_token_address[2:]))
             .all()
         )
         history_pools = {}
         if result is not None:
             for item in result:
-                pool_key = bytes_to_hex_str(item.pool_address)
-                history_pools[pool_key] = UniswapV3Pool(
-                    position_token_address=bytes_to_hex_str(item.position_token_address),
-                    pool_address=pool_key,
-                    token0_address=bytes_to_hex_str(item.token0_address),
-                    token1_address=bytes_to_hex_str(item.token1_address),
-                    factory_address=bytes_to_hex_str(item.factory_address),
-                    fee=item.fee,
-                    tick_spacing=item.tick_spacing,
-                    block_number=item.block_number,
-                    block_timestamp=item.block_timestamp,
-                )
+                pool_key = "0x" + item.pool_address.hex()
+                history_pools[pool_key] = {
+                    "pool_address": pool_key,
+                    "position_token_address": "0x" + item.position_token_address.hex(),
+                    "token0_address": "0x" + item.token0_address.hex(),
+                    "token1_address": "0x" + item.token1_address.hex(),
+                    "fee": item.fee,
+                    "tick_spacing": item.tick_spacing,
+                    "block_number": item.block_number,
+                }
+
     except Exception as e:
+        print(e)
         raise e
     finally:
         session.close()
 
     return history_pools
 
 
+def update_exist_pools(
```

Comment:
This function actually discover new pools

Reply:
rename done